src/components/AuthRootView/index.tsx

```diff
@@ -17,24 +17,25 @@ interface IProps {
 
 const AuthRootView: FC<IProps> = ({ backgroundTitle, children }) => (
   <SafeAreaView style={styles.root}>
-    <TouchableWithoutFeedback style={styles.root} onPress={Keyboard.dismiss} accessible={false}>
-      <View style={styles.root}>
-        <View style={styles.backgroundTextContainer}>
+    <Pressable
```

Comment:
Pressable API is more future-proof over touchable APIs

Reply:
Awesome, I didn't know that. Thanks for this! How come?

Reply:
Just what the official docs say:

https://reactnative.dev/docs/touchablewithoutfeedback